### PR TITLE
add support for module alias discarded name

### PR DESCRIPTION
**Pull request by darky**
```
Failures:

  1) glance_test.import_with_underscore_alias_test
     Failure: ?assertMatch({ ok , _ }, A)
       expected: = { ok , _ }
            got: {error,{unexpected_token,as,{position,27}}}
     output:
```

### CODE REVIEW

File: src/glance.gleam [hunk 1]
```diff
@@ -525,6 +525,10 @@ fn module_name(name: String, tokens: Tokens) -> Result(#(String, Tokens), Error)
 fn optional_module_alias(tokens: Tokens) -> #(Option(String), Tokens) {
   case tokens {
     [#(t.As, _), #(t.Name(alias), _), ..tokens] -> #(Some(alias), tokens)
+    [#(t.As, _), #(t.DiscardName(alias), _), ..tokens] -> #(
+      Some("_" <> alias),
```
Comment (lpil) [inline src/glance.gleam:529]: Let's make this an `AssignmentName` to match the other assignment names which can be discards.

**Review by lpil — COMMENTED** [review]
Awesome, thank you. I've left one comment inline.

Would you mind also updating the changelog? Thank you.

File: src/glance.gleam (hunk 1 above)
Comment (darky) [reply src/glance.gleam]: Fixed, is I properly understanding :)
Changelog updated

**Review by lpil — APPROVED** [review]
Brilliant! Thank you so much!

File: src/glance.gleam (hunk 1 above)
Comment (lpil) [reply src/glance.gleam]: Perfect!

**Pull request merged**